src/studio-header/Avatar.jsx

```diff
@@ -0,0 +1,43 @@
+// This file was copied from edx/frontend-component-header-edx.
```

Comment:
There were no modifications made to this file. It is an exact copy of the corresponding file in the `edx/frontend-component-header-edx` repository.

Reply:
Why do we want to copy? Is it the plan to import this header into `edx/frontend-component-header-edx` so we can remove this later?

Reply:
Are you asking why I want to copy the Avatar component or why I am copying anything from the `edx/frontend-component-header-edx`?

Reply:
The latter. Just want to make sure I understand the process to avoid duplication

Reply:
The Header component that is exported from `edx/frontend-component-header-edx` isn't configurable, unfortunately. I can't control things like the logo that's displayed, nor can I remove the default dropdown items that are displayed (e.g. "Courses", "Programs", and "Discover New").

There is also a [SiteHeader](https://github.com/edx/frontend-component-site-header) component that allows greater configurability. I can change the logo and the menu items. However, I still wouldn't be able to fundamentally change the look of the header in the way that we want. For example, I wouldn't be able to display the course ID as a link to the course in Studio. Only a select few MFEs use the Site Header: [program-console](https://github.com/edx/frontend-app-program-console), [learner-portal-programs](https://github.com/edx/frontend-app-learner-portal-programs),  [learner-portal-enterprise](https://github.com/edx/frontend-app-learner-portal-enterprise), and [learner-portal-base](https://github.com/edx/frontend-learner-portal-base), so it doesn't seem to be in use widely. The rest of the MFEs either use the existing Header component or implement their own.

I spoke to David Joy about what to do ([thread here](https://edx-internal.slack.com/archives/CD4E4BCM9/p1595534564299400), if you're curious), and he recommended taking the pieces that I need from `edx/frontend-component-header-edx`. I think that because the Studio header will eventually diverge significantly from the existing header, it makes to copy it in and make changes to it as necessary now.